blogpost.css

```diff
@@ -0,0 +1,98 @@
+.backdrop{
+	background: black;
+	height: 200px;
+	width: 100%;
+	position: absolute;
+	z-index: 0;
+}
+
+.test{
+	width: 100%;
+	z-index: -1;
+	height:400px;
+	position: absolute;
+}
+
+h1{
+			font-family: 'Barrio';
+			font-size: 80px;
+			color:#fff;
+			text-align: center;
+			position: relative;
+			z-index: 2;
+}
+
+.Announcement{
+	font-family: 'Barrio';
+	font-size: 100px;
+	background-color: #fff;
+	color: #fcce16;
+	border-color: #fcce16;
+	text-align: center;
+	border: 5px dashed;
+}
+
+.Bee{
+	float:center;
+	width:300px;
+	height:200px;
+}
+
+.lyrics{
+	font-family: 'Delius Unicase';
+	font-size: 50px;
+	background-color: #fcce16;
+	text-align: center;
+	color: #fff;
+	width: 1000px;
+	height:auto;
+	padding: 5px 5px 5px 5px;
+}
+
+.main{
+	display: flex;
+	align-items: center;
+	margin-bottom: auto;
+}
+
+.Bee2{
+	height:200px;
+}
+
+.Quote{
+	margin:auto;
+	font-size: 22px;
+	padding: 20px 20px 32px 20px;
+	width: 300px;
+	text-align: center;
+	background-color: #fff;
+	border: 8px solid #fcce16;
+	-webkit-border-radius: 30px;
```

Comment:
these "prefix" things aren't really necessary anymore. Back in the day we needed to use them, but just `border-radius` should be fine now.